flutter/cpp/binary/main.cc

```diff
@@ -331,7 +331,6 @@ int Main(int argc, char* argv[]) {
                       batch_size);
   driver.RunMLPerfTest(mode, min_query_count, min_duration,
                        single_stream_expected_latency_ns, output_dir);
-  LOG(INFO) << "90 percentile latency: " << driver.ComputeLatencyString();
```

Comment:
It was quite useful to use this tool for quick testing and developing of vendor backend. 
Would it be able to parse the latency in `mlperf_driver.h` so we can continue using this API `driver.ComputeLatencyString()`?

Reply:
As far as I remember, loadgen prints a short summary into console.
Is it not sufficient?

Reply:
Yes, you are right. I forget that.